packages/@aws-cdk/aws-sns/package.json

```diff
@@ -20,7 +20,11 @@
         "signAssembly": true,
         "assemblyOriginatorKeyFile": "../../key.snk"
       },
-      "sphinx": {}
+      "sphinx": {},
+      "awsdocs": {
+        "title": "SNS",
+        "tags": ["serverless"]
```

Comment:
Why is this not "data" when `sqs` is ?

Reply:
I feel that SQS is primarily used to ferry data between systems, but SNS is not. But I you feel differently, feel free to update this PR :).

Reply:
Both? "data" and "serverless" both make sense. Forwarding SNS to SQS is a common enough use-case to warrant the "data" tag. It's fan out capability is a useful way to distribute data to many destinations, one of which are "serverless" applications.